### PR TITLE
add el7 mirror support

### PR DESCRIPTION
**Pull request by jonathanspw**
Mostly minor tweaks to deal with doing some stuff a bit different for dead repos.

### CODE REVIEW

File: host_vars/el7.repo.almalinux.org.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  - mirror_name: centos7
+    bootstrap_command:
+      "/usr/bin/rsync -avSH --exclude='.~tmp~' rsync://mirror.facebook.net/centos/TIME
+      {{ mirror_path }}{% if mirror_path_use_subdir %}/centos/{% endif %}"
+    creates: "{{ mirror_path }}{% if mirror_path_use_subdir %}/centos{% endif %}/TIME"
+    rsync_command:
+      "sleep $(((RANDOM\\%3500)+1)) && /usr/bin/flock -n /var/run/centos_rsync.lock-c
+      \"/usr/bin/rsync -avSH --exclude='.~tmp~' --delay-updates
+      --exclude isos/ --exclude atomic/
+      rsync://mirror.facebook.net/centos/{7,7.9*,RPM*}
```
Comment (codyro) [inline host_vars/el7.repo.almalinux.org.yml:13]: What is the reason we're syncing from FB?

File: host_vars/el7.repo.almalinux.org.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      {{ mirror_path }}{% if mirror_path_use_subdir %}/centos/{% endif %}\""
+    rsync_name: centos_rsync
+    rsync_user: root
+    rsync_minute: "0"
+    rsync_hour: "*/1"
+  - mirror_name: centos7-altarch
+    bootstrap_command:
+      "/usr/bin/rsync -avSH --exclude='.~tmp~' rsync://linuxsoft.cern.ch/centos-vault/TIME
```
Comment (codyro) [inline host_vars/el7.repo.almalinux.org.yml:21]: Same as above s/FB/CERN/

(And the rest of these instances)

**Review by codyro — APPROVED** [review]
This is only tangentially related, but when checking this I noticed `rsync.repo.almalinux.org` HTTPS/SSL certificate is expired. We need to see if it's in Zabbix:

```
Thursday, April 25, 2024 at 10:37:35 AM
```

File: host_vars/el7.repo.almalinux.org.yml (hunk 2 above)
Comment (jonathanspw) [reply host_vars/el7.repo.almalinux.org.yml:21]: Official vault doesn't provide rsync.  CERN is a trusted source.

File: host_vars/el7.repo.almalinux.org.yml (hunk 1 above)
Comment (jonathanspw) [reply host_vars/el7.repo.almalinux.org.yml:13]: Official vault doesn't provide rsync.  FB is a trusted source.

**Comment by jonathanspw** [comment]
> This is only tangentially related, but when checking this I noticed `rsync.repo.almalinux.org` HTTPS/SSL certificate is expired. We need to see if it's in Zabbix:
> 
> ```
> Thursday, April 25, 2024 at 10:37:35 AM
> ```

Will be getting fixed in a separate update of the role.  It currently isn't setup to deal with the rsync mirrors but I plan to expand it to them soon.

**Pull request merged**